### PR TITLE
GHA

### PR DESCRIPTION
**Pull request by aricart**
- Removed use of travis for running the tests for the account server
- Updated nats-io deps

### CODE REVIEW

File: .github/workflows/go-test.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        go: [1.16]
```
Comment (kozlovic) [inline .github/workflows/go-test.yaml:8]: Have you tried with 1.17 too?

File: .github/workflows/go-test.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      - name: Run tests
+        shell: bash --noprofile --norc -x -eo pipefail {0}
+        run: |
+          set -e
+          go test -vet=off -i ./...
+          # go test -vet=off -v -run=TestNoRace --failfast -p=1 ./...
+          # coverage via cov.sh disabled while just testing the waters
+          # Also disable race since we are overwhelming the GHA runners.
+          go test -vet=off -v -p=1 --failfast ./...
+          set +e
```
Comment (kozlovic) [inline .github/workflows/go-test.yaml:49]: Not sure if a new line is needed here or not..

File: .gitignore [hunk 3]
```diff
@@ -12,4 +12,5 @@
 *.out
 .idea/
 .vscode/*
-dist/
\ No newline at end of file
+dist/
+vendor/
```
Comment (kozlovic) [inline .gitignore:16]: But if you use go mod now (GO111MODULE=on), then `vendor/` should not be used?

Comment (aricart) [reply .gitignore:16]: removed from the makefile

**Review by kozlovic — APPROVED** [review]
LGTM

**Pull request merged**